chart/values.schema.json

```diff
@@ -2158,12 +2158,12 @@
                                     },
                                     "default": [
                                         {
-                                            "port": "airflow-ui"
+                                            "port": "{{ .Values.ports.airflowUI }}"
                                         }
                                     ],
                                     "examples": [
                                         {
-                                            "port": "sidecar"
+                                            "port": "8080"
```

Comment:
I think I'd rather see the sidecar examples still, or at least use a non-default int port.

Reply:
Yeah.

Reply:
changes done - request you to review the changes @jedcunningham